Assets/Scripts/AnimalMovement.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace AnimalPOV
+{
+    public interface IMovementParameters
+    {
+        float MovementSpeed
+        {
+            get;
+        }
+        float RotationSpeed
+        {
+            get;
+        }
+        float JumpForce
+        {
+            get;
+
+        }
+    }
+    [Serializable()] 
+    public class Rat : IMovementParameters
+        {
+        [SerializeField] 
+        private float movementSpeed = 5f;
+            public float MovementSpeed => movementSpeed;
```

Comment:
Please, write text on one level
```suggestion
        public float MovementSpeed => movementSpeed;
```